policies/optimized-images.md

```diff
@@ -1,8 +1,8 @@
 #  Image Policies Explainer
 
-loonybear@, last updated: 10/30/2018
+loonybear@, last updated: 03/27/2019
 
-<span style="color:#38761d;">Status: Draft Proposal</span>
+<span style="color:#38761d;">Status: in [Origin Trials](https://github.com/GoogleChrome/OriginTrials) in M75</span>
```

Comment:
This is still a proposal, but in Chrome is part of an origin trial... not sure how to make that clear here, but we should make the distinction.